scripts/inventory_check.py

```diff
@@ -0,0 +1,91 @@
+# coding: utf-8
+
+
+import sys
+from ansible.vars.manager import VariableManager
+from ansible.parsing.dataloader import DataLoader
+from ansible.inventory.manager import InventoryManager
+
+
+def parse_inventory(inventory):
+    loader = DataLoader()
+    inv = InventoryManager(loader=loader, sources=[inventory])
+    vars = VariableManager(loader=loader, inventory=inv)
+    all_groups = inv.get_groups_dict()
+    tidb_nodes = all_groups['tidb_servers']
+    tikv_nodes = all_groups['tikv_servers']
+    tidb_servers = {}
+    tikv_servers = {}
+    for tidb in tidb_nodes:
+        var = vars.get_vars(host=inv.get_host(hostname=str(tidb)))
+        ip = var['ansible_host'] if 'ansible_host' in var else var['inventory_hostname']
+        tidb_port = var['tidb_port'] \
+            if 'tidb_port' in var else 4000
+        tidb_status_port = var['tidb_status_port'] \
+            if 'tidb_status_port' in var else 10080
+        deploy_dir = var['deploy_dir']
+
+        if ip in tidb_servers:
+            tidb_servers[ip].append([tidb_port, tidb_status_port, deploy_dir])
+        else:
+            tidb_servers[ip] = [[tidb_port, tidb_status_port, deploy_dir]]
+
+    for tikv in tikv_nodes:
+        var = vars.get_vars(host=inv.get_host(hostname=str(tikv)))
+        ip = var['ansible_host'] if 'ansible_host' in var else var['inventory_hostname']
+        tikv_port = var['tikv_port'] \
+            if 'tikv_port' in var else 4000
```

Comment:
`tikv_port = var.get('tikv_port', 4000)` is more simple and readable,the same below.